code/game/jobs/access.dm

```diff
@@ -318,9 +324,9 @@ proc/get_all_job_icons()
 		if("MRC3") . = size ? "MERC " : "MERC Engineer"
 		else . = paygrade + " " //custom paygrade
 
-#define PAYGRADES_MARINE list("C","E1","E2","E3","E4","E5","E6","E6E","E7","E8","E8E","E9","E9E","O1","O2","O3","O4", "WO")
-#define PAYGRADES_OFFICER list("O1","O2","O3","O4", "WO")
-#define PAYGRADES_ENLISTED list("C","E1","E2","E3","E4","E5","E6","E7","E8","E9")
+#define PAYGRADES_MARINE list("C","E1","E2","E3","E4","E5","E6","E7","E8","E8E","E9","E9E","O1","O2","O3","O4","O5","PO","CPO","WO","CWO","MAJ")
```

Comment:
replace with an autogenerated list based on the paygrades in the marine skill/job datums, if we even need this list anymore that is.